Bonsai.NuGet.Design/IconReader.cs

```diff
@@ -0,0 +1,112 @@
+﻿using System;
+using System.Collections.Concurrent;
+using System.Drawing;
+using System.Drawing.Drawing2D;
+using System.Drawing.Imaging;
+using System.IO;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+using NuGet.Packaging;
+
+namespace Bonsai.NuGet.Design
+{
+    class IconReader
+    {
+        static readonly HttpClient httpClient = new();
```

Comment:
It wasn't in the original implementation, but it seems like it'd be a good idea to enable the HTTP cache here too just like it is in `DocumentationHelper`.

(Compression *could* be enabled too, but it doesn't really matter in this context. NuGet.org won't bother trying to further compress the PNGs anyway, which makes sense.)